doc/source/using-ray-on-a-cluster.rst

```diff
@@ -1,9 +1,9 @@
-Using Ray on a Cluster
-======================
+Manual Cluster Setup
+====================
 
 .. note::
 
-    If you're using AWS you can use the automated `setup commands <http://ray.readthedocs.io/en/latest/autoscaling.html>`__.
+    If you're using AWS or GCE you should use the automated `setup commands <http://ray.readthedocs.io/en/latest/autoscaling.html>`__.
```

Comment:
I think we should be consistent and use `GCP` throughout (as opposed to `GCE`). What do you think? @hartikainen

Reply:
I think GCP is the same for google as AWS is for amazon, and GCE is the same for google as EC2 is for amazon. So here I would use GCP.